code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -229,6 +248,57 @@ proc/issyndicate(mob/living/M as mob)
 /datum/game_mode/proc/random_radio_frequency()
 	return 1337 // WHY??? -- Doohl
 
+/datum/game_mode/proc/equip_sleeper(mob/living/carbon/human/sleeper_agent)
+	if(!istype(sleeper_agent))
+		return
+	update_synd_icons_added(sleeper_agent.mind)
+
+	if(sleeper_agent.mind)
+		if(sleeper_agent.mind.assigned_role == "Clown")

```

Comment:
Probably check if they have the clumsy gene rather than if their role is clown here? There are some ways you could become clumsy without being a clown

Reply:
I stole those checks from the autotraitor code, who does it the same way. I suppose I could change it, but in both places at once.

Reply:
Ahh okay. In that case I'd probably do what it does in autotraitor code then, to make things consistent